### ISSUE TITLE
Take internal order for multishop "all shop" section

### ISSUE THREAD

**Issue by pollat** [PrestaShop/PrestaShop#15158]
On the back office of prestashop multishop, we can take a internal order only if we choice a specific shop, but not if we are on the "all shop" section: on the order page of "all shops" section missing the "+" to take a internal order (see image attached). 

So, add a "+" to take internal order on the "All shops" section and, on first step after, the possibility to choice what shop use to take our order. 

Most of people that use multishop, menage the orders only on "all shop" section (that have all orders of all shops) and if missing the "+" doesn't discover the possibility to take an internal order. 

![take-order](https://user-images.githubusercontent.com/9538665/63320219-5770f800-c31d-11e9-8569-c32d7de4cc0c.jpg)

**Comment by khouloudbelguith** [comment]
Hi @pollat,

This is normal behavior.
You have to select a shop before creating new orders.
![image](https://user-images.githubusercontent.com/16067358/63329608-2a771200-c32a-11e9-860a-4625b211d0b3.png)

Thanks for your understanding!

**Comment by pollat** [comment]
@khouloudbelguith, yes I know, but this a request to add this feature also on "All shop" section and then select the shop on the first step of order setting.

**Comment by khouloudbelguith** [comment]
@pollat, thanks for your feedback.
Ping @marionf @colinegin what do you think?

Thanks!

**Comment by boo-code** [comment]
Measured on 9.2.x. The discoverability half of this is resolved; the feature itself is still refused, and deliberately.

The button is no longer absent outside a single-shop context. `OrderController::getOrderToolbarButtons()` always renders "Add new order", and when the shop constraint is not a single shop it marks that button `disabled`, points its href at `#`, and attaches the help text `You can use this feature in a single-store context only. Switch contexts to enable it.` So the affordance is visible and explains itself, which is what the report was really about when it noted that a missing "+" hides the feature from merchants who work in "All shops".

Creating from that context is still refused, and on the server rather than only in the interface: the create action tests the same single-shop constraint and, when it does not hold, flashes `You have to select a shop before creating new orders.` and redirects back to the list.

What remains of the request is therefore its second half - choosing the shop as a step of the creation flow instead of before entering it - and that boundary is currently asserted in two places, the toolbar and the controller.
